README.md

```diff
@@ -73,5 +73,32 @@ if ($webhookEvent->getType() === WebhookEvent::TYPE_PARCEL_STATUS_CHANGED) {
 }
 ```
 
+### Retieve a list of service points
+
+```php
+use JouwWeb\Sendcloud\Client;
+use JouwWeb\Sendcloud\Exception\SendcloudRequestException;
+
+// Sendcloud uses another api url for service points
+// if you use the default one (i.e https://panel.sendcloud.sc/api/v2/), the API will return a 404 error
+$client = new Client('your_public_key', 'your_secret_key', null, Client::SERVICE_POINTS_BASE_URL);
```

Comment:
I think we can fix this more transparently and intuitively by always using absolute URLs instead of relative paths to call the API. Configuring a base URI on the client only makes sense when there is just one =)

So we can remove the `base_uri` argument in `Client::__construct()` and add arguments and properties for two base URIs `$apiBaseUrl` and `$servicePointsBaseUri`. 

Another alternative is to create a second `ServicePointsClient` because it's essentially a different API, but that sounds like needless complication.

Please let me know your opinion about this and whether you'd like me to help with the changes!

Reply:
I'm all out to make this a lot simpler to use too :smile: 

 > Another alternative is to create a second ServicePointsClient because it's essentially a different API, but that sounds like needless complication.
 
 I already opened an issue (#34) with this extract alternative if you want to take a look :)

Reply:
=O Maybe I remembered that subconsciously but it must be a great idea in that case.

I'll have a swing at building that on top of this PR so we can release it in one go.